src/jdaTemplate/app/src/main/java/bot/SlashCommandListener.java

```diff
@@ -0,0 +1,17 @@
+package bot;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class SlashCommandListener extends ListenerAdapter {
+    @Override
+    public void onSlashCommand(SlashCommandEvent event) {
+        try {
+            App.getCommands().get(event.getName()).run(event);
+        } catch (Exception e) {
+            event.reply("An unknown error occurred while executing the command.").queue();
+
+            e.printStackTrace();
```

Comment:
It is probably a good idea to use SLF4J to log information since JDA uses SLF4J as well. You can obtain a logger using `LoggerFactory.getLogger(this.getClass())` (normally saved in an instance variable) and then call your logger like `logger.error("An unknown error occurred while executing the command.",e)`. This would print a nice logging message.